CHANGES.rst

```diff
@@ -5,7 +5,8 @@ Changelog of freq
 0.3 (unreleased)
 ----------------
 
-- Nothing changed yet.
+- Makes IFRAME_BASE_URL a setting.
+- Fixes harmonics (harmonics 1 is correct again)
```

Comment:
Wordt dit hier gedaan? Je verandert niet iets. Mist er wat?

Je hoogde een versie van een pakketje op in de site: misschien hier dan een minimale versie daarvoor in de `setup.py` aangeven?